### DISCUSSION TITLE
Can tsc-alias be used to permanently change paths to relative path in the rootDir?

### DISCUSSION THREAD

**Discussion by gablabelle** [justkey007/tsc-alias#27] [Q&A] [answered]
Hello,

Can `tsc-alias` be used to permanently change paths to relative path in the `rootDir`? (not only in the `outDir`)

I'm currently migrating a few repos to a single monorepo and would like to get rid of these paths.

If not, any other tool to suggest that can help do that?

**Comment by justkey007** [comment, marked as answer]
Hello,
No it is not possible because it is not its goal. But you can fork and modify that as you wish in order to affect the .ts files.

**Reply by gablabelle** [reply]
OK thanks for the quick reply @justkey007

**Reply by raouldeheer** [reply]
> OK thanks for the quick reply @justkey007

@gablabelle Can this question be marked as answered?
